PSI_Tracker.js

```diff
@@ -0,0 +1,645 @@
+      if(api == "PSI" && save_screenshot == true){
+        ScriptApp.newTrigger('runBatchPSIWithScreenshots')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      else if(api == "PSI" && save_screenshot == false){
+        ScriptApp.newTrigger('runBatchPSI')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      else if(api == "CrUX"){
+        ScriptApp.newTrigger('runBatchCrUX')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      else if(api == "CrUX History"){
+        ScriptApp.newTrigger('runBatchCrUXHistory')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      else if(api == "Green Domain"){
+        ScriptApp.newTrigger('runBatchGreenDomain')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      else if(api == "Accessibility"){
+        ScriptApp.newTrigger('runBatchAccessibility')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      else if(api == "Sustainability"){
+        ScriptApp.newTrigger('runBatchSustainability')
+          .timeBased()
+          .everyMinutes(TIME_BETWEEN_BATCHES)
+          .create();
+      }
+      runBatch(api, save_screenshot);
+    }
+}
+
+
+const COLUMN_STATUS = 8;
+
+/**
+ * Sets a clear state for the program, changing the "Status" column of all valid URLs to "⏳", waiting to be parsed
+ */
+function initURLStatus() {
+    toast("Initialising","Removing previous executions and setting a blank state");
+    removeBatchTriggers();
+    let sheet = CONFIG_SHEET;
+    let last_row = sheet.getLastRow();
+    let last_column = sheet.getLastColumn();
+    let values = sheet.getRange(1, 1, last_row, last_column).getValues();
+    // Go line by line of all URLs and change value and note
+    for (let i = 1; i < values.length; i++) {
+        const url = values[i][4];
+        const active = values[i][7]; //G
+        sheet.getRange(i+1,COLUMN_STATUS+1).setValue("")
+        sheet.getRange(i+1,COLUMN_STATUS+1).setNote("")
+        // Skip empty lines
+        if(url == "" || url == null || url == undefined){continue;}
+        if(!active){continue;}
+        sheet.getRange(i+1,COLUMN_STATUS+1).setValue("⏳")
+    }
+    SpreadsheetApp.flush();
+}
+
+/**
+ * Run Batch, reads the number of URLs to send to the API and runs a batch of them
+ */
+function runBatch(api = "PSI", save_screenshot = false){
+    let sheet = CONFIG_SHEET;
+    let NUMBER_OF_URLS_PER_BATCH = +sheet.getRange("B21").getValue();
+    let last_row = sheet.getLastRow();
+    let last_column = sheet.getLastColumn();
+    let values = sheet.getRange(1, 1, last_row, last_column).getValues();
+
+    // STEP 1: Create a batch of NUMBER_OF_URLS_PER_BATCH URL
+    let batch = [];
+    for (let i = 1; i < values.length; i++) {
```

Comment:
A comment that you're skipping the first row would be helpful

Reply:
on it!